gadgets/trace_lsm/gadget.yaml

```
@@ -0,0 +1,638 @@
+name: trace lsm
+description: a strace for LSM tracepoints
+homepageURL: https://inspektor-gadget.io/
+documentationURL: https://inspektor-gadget.io/docs
+sourceURL: https://github.com/inspektor-gadget/inspektor-gadget/
+tracers:
+  lsm:
+    mapName: events
+    structName: event
+structs:
+  event:
+    fields:
+    - name: mntns_id
+      description: Mount namespace inode id
+      attributes:
+        template: ns
+    - name: pid
+      attributes:
+        template: pid
+    - name: tid
+      attributes:
+        template: pid
+    - name: uid
+      attributes:
+        template: pid
+    - name: gid
+      attributes:
+        template: pid
+    - name: tracepoint
+      description: lsm tracepoint enum
+      attributes:
+        width: 16
+        alignment: left
+        ellipsis: end
+    - name: comm
+      description: command
+      attributes:
+        template: comm
+    - name: timestamp
+      attributes:
+        template: timestamp
+ebpfParams:
+  trace_all:
+    key: trace_all
+    defaultValue: ""
+    description: trace all LSM tracepoints
+  trace_binder_set_context_mgr:
+    key: trace_binder_set_context_mgr
```

I suggest to use `trace-%s` for the CLI flags:
```suggestion
    key: trace-binder_set_context_mgr
```

Ditto below.